54.md

```diff
@@ -0,0 +1,60 @@
+NIP-54
+======
+
+Topical Notes
+-------------
+
+`draft` `optional`
+
+This NIP enables authors to post exclusively into topics, without polluting their main feed. Authors can post to locations, hashtags, labels, and urls in a way that only followers of those topics will receive their posts. 
+
+## Topical Note Kind
+
+Kind `21` defines a `kind:1`-equivalent event that MUST NOT be displayed unless the user follows one of the `g`, `t`, `r` or `l` tags. 
+
+```jsonc
+{
+  "kind": 21,
+  "tags": [
+    ["g", "<geohash>"],
+    ["t", "<hashtag>"],
+    ["r", "<url>"],
+    //.. 
+    ["L", "<label namespace>"],
+    ["l", "<label>", "<label namespace>"],
+  ],
+  "content": "This is a note for followers of the hashtag, location OR url. My followers should not see this.",
+  //...
+}
+```
+
+`.content` MUST be plain text and tags that mention, quote and assemble threading structures MUST follow [NIP-10](10.md).
+
+## Topical Relay Setup
+
+Kind `30021` defines a topic follower. It registers a list of relays to aggregate events for each topic. 
+
+The `d` tag of this kind is equal to `name:value` of the topics used in kind `21`. 
```

Comment:
```suggestion
The `d` tag of this kind is equal to `tag:value` of the topics used in kind `21`. 
```

How do we handle multiple topics here? Comma separated?

Reply:
I just replaced the name:value for a UUID and added the following tags directly on the 30021 event as well.